src/main/java/org/graylog/integrations/aws/resources/requests/AWSInputCreateRequest.java

```diff
@@ -29,15 +29,12 @@
 @JsonAutoDetect
 @AutoValue
 @WithBeanGetter
-public abstract class AWSInputCreateRequest {
+public abstract class AWSInputCreateRequest implements AWSRequest {
```

Comment:
This implementation of the `AWSRequest` interface ensures that the key/secret/region fields are all specified. Most AWS API request objects implement this.

Reply:
@kyleknighted We can help to cherry-pick this in to any branch that you are currently working on once merged.